client/ayon_core/hosts/maya/plugins/publish/extract_fbx_animation.py

```diff
@@ -35,7 +35,8 @@ def process(self, instance):
         fbx_exporter = fbx.FBXExtractor(log=self.log)
         out_members = instance.data.get("animated_skeleton", [])
         # Export
-        instance.data["constraints"] = True
```

Comment:
@moonyuet was the removal of constraints being `True` still intentional? I thought you mentioned you were to revert that?

Reply:
yes, it is.